padl/transforms.py

```diff
@@ -931,6 +997,16 @@ def _pd_direct_subtransforms(self) -> Iterator[Transform]:
             if isinstance(v, Transform):
                 yield v
 
+    def _pd_get_non_target_stage_idx(self, stage_used):
```

Comment:
I would appreciate documentation for these three functions `_pd_get_non_target_stage_idx`, `_pd_get_target_stage_idx`, and `_pd_get_error_idx` even though they are hidden. It's not clear to me what they are for.

Reply:
- [ ] Also consider adding types to the arguments

Reply:
ok, I'll do this